go/api/sorted_set_commands_test.go

```diff
@@ -526,3 +526,43 @@ func ExampleGlideClient_ZUnionWithScores() {
 	// Output:
 	// map[one:1 three:3 two:5.5]
 }
+
+func ExampleGlideClient_ZLexCount() {
```

Comment:
Consider adding multiple examples. The integration tests offer a couple examples with simple setup that can be easily refactored into examples. The goal is to make it easy on the developer so they don't have to download source code and dig around in our test files just to figure out how to use the client.